app/services/force/event_subscriber_translate_service.rb

```diff
@@ -110,12 +110,30 @@ def translate_and_cache(event)
       )
     end
 
-    def translate_event_values(values)
-      logger("Translating values: #{values.inspect}")
+    def process_event_values(listing_id, values)
       return [] unless values&.values
 
+      listing = Request.new(parse_response: true).cached_get(
+        "/ListingDetails/#{CGI.escape(listing_id)}", nil, true
+      ).first
```

Comment:
question: This will update the cache for the listing, will there be any weird interactions with the behavior getting implemented in this PR: https://github.com/SFDigitalServices/sf-dahlia-web/pull/2283 ?

Also, I'm a bit apprehensive about mixing concerns, i.e. updating the Listing cache from within this service.

Reply:
Yeah good point - we can just not do the force and just get whatever is cached